spid_cie_oidc/onboarding/tests/token_request_settings.py

```diff
@@ -2,7 +2,7 @@
 
 TOKEN_REQUEST = {
     "client_id": "https://rp.cie.it",
-    "client_assertion": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiw...",
+    "client_assertion": "head.payload.verifysignature",
```

Comment:
we used to call this __private_key_jwt__ that's the client authentication method required to this endpoint